File: AllMyBooksArePacked/class_utils/Book.js
const _ = require('underscore');

class Book {
	constructor(book) {
		_.extend(this, {
			'title': book.title,
			'isbn-10': book.isbn,
			'price': book.price,
			'shipping_weight': book.weight,
			'author': book.author,
			'_weight': parseFloat(book.weight)
		});
	}
};

module.exports = Book;

File: AllMyBooksArePacked/lib/book.rb
module Items
  class Book
    attr_reader :author, :title, :weight, :isbn
    attr_accessor :price

    def initialize(title, author, price, isbn, weight)
      @author = author
      @title = title
      @price = price
      @isbn = isbn
      @weight = weight
    end

    def properties
      [@title, @author, @price, @isbn, @weight]
    end 
  end
end

